pennylane/_qubit_device.py

```diff
@@ -535,14 +535,14 @@ def states_to_binary(samples, num_wires, dtype=np.int64):
         This is an auxiliary method to the generate_samples method.
 
         Args:
-            samples (List[int]): samples of basis states in base 10 representation
+            samples (ndarray): integer samples of basis states in base 10 representation
```

Comment:
Not really a change request, but it seems like we're not consistent in using `ndarray`, `np.ndarray` or `numpy.ndarray` at all. 🤔

Reply:
Fair point! Was also considering this. We have [`array` mentioned in our guide](https://pennylane.readthedocs.io/en/stable/development/guide/documentation.html#functions-and-methods), changed occurrences in the file to that.